.github/workflows/gauntlet.yml

```diff
@@ -61,5 +61,18 @@ jobs:
         with:
           name: chainlink-cosmos
           authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
+      - name: Cache Wasmd Artifacts
```

Comment:
cache the root dir "artifacts/" folder which contains the optimized .wasm bytecode files. This usually takes a while to compile